components/exercises/chapter-5-oscar/Everything/some.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { some } from './some';
+
+describe('some', () => {
+  it('should return true if each satisfies the condition', () => {
+    const arr = [2, 4, 6, 8];
+    const callBack = (num: number) => num < 10;
+    expect(some(arr, callBack)).toBe(true);
+  });
+
+  it('should return false if at least one element satisfies the condition', () => {
+    const arr = [2, 4, 7, 8, 11];
+    const callBack = (num: number) => num < 10;
+    expect(some(arr, callBack)).toBe(false);
+  });
```

Comment:
Can we add tests for arrays that contain other elements apart from numbers? Like strings, objects, arrays, null, undefined....

Reply:
test added for every case